amundsen_application/static/js/components/TableDetail/ColumnListItem/index.spec.tsx

```diff
@@ -132,7 +132,113 @@ describe('ColumnListItem', () => {
       expect(newWrapper.find(ColumnStats).exists()).toBe(true);
     });
 
-    it('renders EditableSection with correct edit text and Url when expanded', () => {
+    it('renders EditableSection with correct description, edit text and Url when expanded', () => {
```

Comment:
Nit: I would create a describe clause `describe('when expanded', () => ...` and nest your test and the previous one inside of it.